io.github.JakubMelka.Pdf4qt.yml

```diff
@@ -0,0 +1,28 @@
+app-id: io.github.JakubMelka.Pdf4qt
+runtime: org.kde.Runtime
+runtime-version: 5.15-22.08
+sdk: org.freedesktop.Sdk
+command: pdf4qt
+finish-args:
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --filesystem=home
+  - --share=network
+modules:
+  - name: openjpeg
+    buildsystem: cmake-ninja
+    sources:
+      - type: archive
+      - url: https://github.com/uclouvain/openjpeg/archive/refs/tags/v2.5.0.tar.gz
+      - sha256: 0333806d6adecc6f7a91243b2b839ff4d2053823634d4f6ed7a59bc87409122a
+  - name: lcms
+    sources:
+      - type: archive
+        url: https://github.com/mm2/Little-CMS/archive/refs/tags/lcms2.15.tar.gz
+        sha256: 30d928c9b564c670b6b57c917e5c2599247b713ba6cf2dbd089a8efa70a1952f
+    - name: pdf4qt
```

Comment:
this indentation is incorrect.

you chose to use yaml, you must make sure your indentations are consistent.

Reply:
I'll fix that right now. I'll double check by running it through a yaml linter.